gantry/firmware/CMakeLists.txt

```diff
@@ -215,17 +215,6 @@ add_custom_target(gantry-debug-y
         )
 
 
+target_stm32G4_flash(gantry-x $<TARGET_FILE_DIR:gantry-x>/gantry-x.bin $<TARGET_FILE_DIR:bootloader-gantry-x>/bootloader-gantry-x.bin)
```

Comment:
does this properly pick up the dependency of building the binary firmware files? it seems a bit awkward to haveto use the generator like this

Reply:
It didn't pick up the binary dependencies correctly. 

I've rewritten this without a function.